sys-libs/cdefs-standalone/files/sys/cdefs.h

```diff
@@ -0,0 +1,40 @@
+// taken from https://github.com/void-linux/void-packages/blob/master/srcpkgs/musl-legacy-compat/files/cdefs.h
```

Comment:
honestly I'd prefer it if we made a git repo for each of these (same rationale as for tree except for size)

Reply:
sure